buildroot-external/board/arm-uefi/generic-aarch64/kernel.config

```diff
@@ -12,6 +12,18 @@ CONFIG_VIRTIO_VSOCKETS=m
 CONFIG_VIRTIO_MMIO=y
 CONFIG_SCSI_VIRTIO=y
 CONFIG_HW_RANDOM_VIRTIO=y
+CONFIG_VIRTIO_MMIO_CMDLINE_DEVICES=y
+CONFIG_VIRTIO_IOMMU=m
+CONFIG_CRYPTO_ENGINE=m
+CONFIG_CRYPTO_DEV_VIRTIO=m
+
+# These options are needed for virtio-fs
+CONFIG_FUSE_FS=y
+CONFIG_VIRTIO_FS=y
+CONFIG_DAX=y
+CONFIG_FS_DAX=y
+CONFIG_DAX_DRIVER=y
```

Comment:
```suggestion
```
We can remove this, it was removed in kernel >5.16 ([ref1](https://patchwork.kernel.org/project/linux-nvdimm/patch/20211109083309.584081-4-hch@lst.de/), [ref2](https://cateee.net/lkddb/web-lkddb/DAX_DRIVER.html)).